src/fides/api/schemas/saas/shared_schemas.py

```diff
@@ -45,6 +45,12 @@ class IdentityParamRef(BaseModel):
     identity: str
 
 
+class DataSetRef(BaseModel):
```

Comment:
i dont think this is a good name. Technically a reference to any param value will work, not just dataset reference values.

Reply:
same thing for the field dataset_reference.